app/ide-desktop/lib/dashboard/src/configurations/inputBindings.ts

```diff
@@ -41,8 +41,8 @@ export interface DashboardBindingNamespace extends ReturnType<typeof createBindi
 export type DashboardBindingKey = keyof typeof BINDINGS
 
 /** Create a keybind and mousebind namespace. */
-export function createBindings() {
-  return inputBindings.defineBindingNamespace('dashboard', BINDINGS)
+export function createBindings(register = true) {
+  return inputBindings.defineBindingNamespace('dashboard', BINDINGS, register)
```

Comment:
Instead of adding a parameter here, I think this function should simply never do the registration. That simplifies the code a bunch.

Reply:
makes sense. i added it because i figured users may forget to register the namespace - but definitely agree that it does complicate the code a lot